fdbserver/BackupWorker.actor.cpp

```diff
@@ -561,7 +561,7 @@ ACTOR Future<Void> monitorBackupProgress(BackupData* self) {
 		// check all workers have started by checking their progress is larger
 		// than the backup's start version.
 		state Reference<BackupProgress> progress(new BackupProgress(self->myId, {}));
-		wait(getBackupProgress(self->cx, self->myId, progress));
+		wait(getBackupProgress(self->cx, self->myId, progress, /*logging=*/false));
```

Comment:
A knob can make it more flexible, in case we need to turn the logging on in prod.